### ISSUE TITLE
Video is no longer playing

### ISSUE THREAD

**Issue by slavajacobson** [m90/seeThru#52]
Is it related to Chrome updated? The video on the demo page isn't working either: http://m90.github.io/seeThru/moving-alpha/

**Comment by m90** [comment]
Yeah, console says so:

```
Uncaught (in promise) DOMException: play() failed because the user didn't interact with the document first. https://goo.gl/xX8pDD
```

I recently encountered the same issue when updating the automated tests for this script and has to start Chrome using `--autoplay-policy=no-user-gesture-required`, see: https://github.com/m90/seeThru/blob/2c43f32c364f59fa584d2811484f099dd689ffb1/karma.conf.js#L62-L67

Unfortunately I am not sure what `seeThru` can do about this other than warn users in README and demos. Do you have any other idea about where to start?

**Comment by m90** [comment]
I released v4.0.0 with removed "autoplay" features, so the caller is now responsible for handling behavior like this (and can thus use approaches like described above).
